Sources/Plot/API/ControlFlow.swift

```diff
@@ -26,8 +26,11 @@ public extension Node {
     /// be created.
     /// - parameter optional: The optional value to unwrap.
     /// - parameter transform: The closure to use to transform the value into a node.
-    static func unwrap<T>(_ optional: T?, _ transform: (T) throws -> Node) rethrows -> Node {
-        try optional.map(transform) ?? .empty
+    /// - parameter defaultValue: The closure to use when the value is nil.
```

Comment:
Please update the documentation to reflect the new implementation. Also, if you could again keep the API consistent with `if`, I'd really appreciate it. So name this parameter `fallbackNode` rather than `defaultValue`.